mwdb/core/log.py

```diff
@@ -16,12 +33,26 @@ def filter(self, record):
 # Don't propagate to root logger
 logger.propagate = False
 
-# Setup JSON stream handler for main logger
+# Setup stream handler for main logger
 handler = logging.StreamHandler()
-handler.setFormatter(
-    logmatic.JsonFormatter(fmt="%(filename) %(funcName) %(levelname) %(lineno) %(module) %(threadName) %(message)"))
-logger.addHandler(handler)
+
+if logging_type == "standard":
+    formatter = logging.Formatter(fmt="File:%(filename)s - Function:%(funcName)s - Level:%(levelname)s - Line:"
+                                      "%(lineno)s - Module:%(module)s - Thread:%(threadName)s" 
+                                      " - Message:%(message)s - %(extra)s")
+    handler.setFormatter(
+        formatter
+    )
+
+elif logging_type == "json":
+    formatter = logmatic.JsonFormatter(fmt="%(filename) %(funcName) %(levelname) %(lineno) %(module) %(threadName) %(message)")
+
+    handler.setFormatter(
+        formatter
+    )
```

Comment:
This one is common for both conditions.
```suggestion
handler.setFormatter(formatter)
```